static/script.js

```diff
@@ -66,8 +65,9 @@ function updateData(){
 
 	// CAPTURE USER INPUT FOR ANALYSIS TYPE SELECTION
 	var checked = document.getElementById("interpolation").checked
-
+	var checkedheat = document.getElementById("heatmap").checked
 	// CAPTURE USER INPUT FOR HEAT MAP 'SPREAD' OR OTHER PARAMETERS
+	var spreadheat = document.getElementById("spread").value
 
 	// SEND USER CHOICES FOR ANALYSIS TYPE, CELL SIZE, HEAT MAP SPREAD, ETC. TO SERVER
 	request = "/getData?lat1=" + lat1 + "&lat2=" + lat2 + "&lng1=" + lng1 + "&lng2=" + lng2 + "&w=" + w + "&h=" + h + "&cell_size=" + cell_size + "&analysis=" + checked
```

Comment:
you should modify the request to send the status of the checkedheat variable to the server